btx/diagnostics/run.py

```diff
@@ -429,6 +454,77 @@ def visualize_energy_stats(self, output=None, outlier_threshold=-np.inf):
             if output is not None:
                 fig.savefig(output, dpi=300, bbox_inches='tight')
     
+    def visualize_stats_interactive(self, y_key='mean', color_by='beam_energy_eV'):
+        """
+        Plot trajectories of run statistics, interactively.
+
+        Parameters
+        ----------
+        y_key : str
+            self.stats_final key to plot
+        color_by : str
+            self.stats_final key used to color
+        """
+        hv.extension('bokeh')
+
+        y = self.stats_final[y_key]
+
+        df = pd.DataFrame({
+            'evt_id' : np.arange(y.shape[0]), 
+            'y' : self.stats_final[y_key], 
```

Comment:
`self.stats_final[y_key]`  (line 474) could be replaced by the variable `y` since it's already been defined. (Line 470)

Reply:
good point